.classpath

```diff
@@ -7,5 +7,6 @@
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="lib" path="/Users/jake/wpilib/user/java/lib/CTRLib.jar"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```

Comment:
Is that supposed to be a variable? Shouldn't it be surrounded in braces or brackets or something? Anyway, this is probably fine.